packages/contracts/test/lib/libAsset.spec.ts

```diff
@@ -33,7 +32,7 @@ describe("LibAsset", function() {
     loadFixture = createFixtureLoader([wallet, other]);
   });
 
-  beforeEach(async function() {
+  beforeEach(async function () {
```

Comment:
Generally we dont have to test things that aren't explicitly code written by us (i.e. reverting if the `assetId` is not an address. But, I also think the more tests the better so I'm happy to leave it

Reply:
yeah we don't really need it, going to clean it up.